src/EnergyPlus/TARCOGDeflection.cc

```diff
@@ -44,9 +44,6 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-// C++ Headers
-#include <cmath>
-
```

Comment:
:+1:

Reply:
Cheers for CLion that sees these things :)